packages/api/endpoints/logs.js

```diff
@@ -1,8 +1,17 @@
 'use strict';
 
 const router = require('express-promise-router')();
+const log = require('@cumulus/common/log');
 const { Search } = require('../es/search');
 
+function convertLogLevelForQuery(query) {
+  if (!query.level) {
+    return query;
+  }
+
+  return Object.assign({}, query, { level: log.convertLogLevel(query.level) });
```

Comment:
So before, the endpoint was taking `req.query.level` as a number, but now it takes `req.query.level` as a string and returns a number from `log.convertLogLevel`? So the benefit is people can pass the string for the log level that they want, which they are more likely to know, instead of the number?

Reply:
yes, exactly that. The dashboard is actually passing it up as a string now and I think it's way more clear to pass it up as a string.

Reply:
Also, this could really go either way and is a judgment call so I'm definitely open to arguments either way.

Reply:
I think it's fine, I just wanted to make sure I understood